backend/src/test/java/com/example/backend/controller/NgoControllerTest.java

```diff
@@ -0,0 +1,198 @@
+package com.example.backend.controller;
+
+import com.example.backend.dto.NgoDto;
+import com.example.backend.model.Ngo;
+import com.example.backend.repository.NgoRepo;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.test.web.reactive.server.WebTestClient;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class NgoControllerTest {
+
+    @LocalServerPort
+    private int port;
+
+    @Autowired
+    private WebTestClient testClient;
+
+    @Autowired
+    private NgoRepo ngoRepo;
+
+    @BeforeEach
+    public void cleanUp() {
+        ngoRepo.deleteAll();
+    }
+
+    @Test
+    void getNgos_ReturnNgosInDatabase() {
+
+        //GIVEN
+        Ngo test1 = Ngo.builder().name("New NGO").email("test@mail.org").url("https://test.com").tagline("Test tagline").description("Another NGO in the database").sdg("15").image("https://01.png").city("Hamburg").country("Germany").region("Europe").build();
```

Comment:
Schwer zu lesen. Besser 
```suggestion
        Ngo test1 = Ngo.builder().name("New NGO")
        .email("test@mail.org")
        .url("https://test.com")
        .tagline("Test tagline")
        .description("Another NGO in the database")
        .sdg("15")
        .image("https://01.png")
        .city("Hamburg")
        .country("Germany")
        .region("Europe").build();
```
Mach das am besten in Intellij, dann ist die Einrückung gleich richtig.
Im Weiteren könnte man das immer machen ;) .